rescript/subsample.py

```diff
@@ -0,0 +1,25 @@
+# ----------------------------------------------------------------------------
+# Copyright (c) 2020, QIIME 2 development team.
+#
+# Distributed under the terms of the Modified BSD License.
+#
+# The full license is in the file LICENSE, distributed with this software.
+# ----------------------------------------------------------------------------
+
+import random
+from q2_types.feature_data import DNAIterator
+
+
+def subsample_fasta(sequences: DNAIterator,
+                    subsample_size: float = 0.1,
+                    random_seed: int = 42) -> DNAIterator:
```

Comment:
Probably a good idea to set the seed to either a typical value like `1`, `123`, or `999`, or something. Although I personally like `42` the answer to the meaning of life, the universe, and everything... it'll be lost on most. :-)

Reply:
Oooops, sorry for that! I had it changed already but then must have CMD+Zed too far 😅  - will change again!